addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -84,6 +86,7 @@ export class DateTimeField extends Component {
                 value,
                 type: this.field.type,
                 range: this.isRange(value),
+                showSeconds: this.props.showSeconds,
```

Comment:
I would not add this as pickerProps (not relevant for the picker). It's rather a hookParam (argument to pass next to pickerProps to the call to useDateTimePicker